src/content/configuration/externals.md

```diff
@@ -330,3 +331,60 @@ module.exports = {
   externalsType: 'promise'
 };
 ```
+
+### `script`
+
+External script can be loaded from any url when `externalsType` is set to `script`.
+
+There're two syntaxes for loading external script in webpack configuration:
+
+1. `{externals: {packageName: ['http://example.com/script.js', 'global', 'property', 'property']}}` (properties are optional)
```

Comment:
These example snippets are less readable than for example just showing the syntax in the code snippet. Also properties are not described well. I suggest you do two full config examples for each of the syntaxes with more details on `property`, `property` use-cases and usage.